doc/decisions/decision_process.md

```diff
@@ -74,9 +99,11 @@ The first step is to create a PR with:
 - **one** decision, where at least the "Problem" and "Considered Alternatives" are filled out.
```

Comment:
What if there are no considered alternatives yet?

Reply:
Yes, you are right, I changed in 190cec69746f77ed9824bce3e01397076bbc1f6a